test/client/http_service.test.js

```diff
@@ -0,0 +1,312 @@
+const MockServer = require('../fixtures/mock_server');
+const StitchMongoFixture = require('../fixtures/stitch_mongo_fixture');
+
+import { buildClientTestHarness, extractTestFixtureDataPoints } from '../testutil';
+
+describe('Executing http service functions', () => {
+  const server = new MockServer();
+  const test = new StitchMongoFixture();
+
+  beforeAll(() => test.setup());
+  afterAll(() => test.teardown());
+
+  const SERVICE_TYPE = 'http';
+  const SERVICE_NAME = 'gd2';
+
+  let th;
+  let service;
+  let serviceId;
+
+  beforeEach(async() => {
+    const { apiKey, groupId, serverUrl } = extractTestFixtureDataPoints(test);
+    th = await buildClientTestHarness(apiKey, groupId, serverUrl);
+
+    const httpService = await th
+      .app()
+      .services()
+      .create({ type: SERVICE_TYPE, name: SERVICE_NAME });
+
+    service = th.stitchClient.service(SERVICE_TYPE, SERVICE_NAME);
+    serviceId = httpService._id;
+  });
+
+  afterEach(async() => th.cleanup());
```

Comment:
nit/note: you actually don't need `async` here, since you're not relying on using `await` anywhere in the function body. Plus since the function returns a promise, jest will wait for it to resolve anyway